src/corona_hakab_model/main.py

```diff
@@ -102,6 +132,56 @@ def main():
     plt.show()
 
 
+def generate_command(args):
+    if args.consts_folder:
+        generate_from_folder(args.consts_folder)
+        return
+    elif args.master_folder:
+        generate_from_master_folder(args.master_folder)
+        return
+
+    if not args.output_folder:
+        logger.error("No output folder given! use --output-folder")
+
+    if args.circles_consts_path:
```

Comment:
extract this part to function
`    if args.circles_consts_path:
        circles_consts = CirclesConsts.from_file(args.circles_consts_path)
    else:
        circles_consts = CirclesConsts()

    if args.matrix_consts_path:
        matrix_consts = MatrixConsts.from_file(args.matrix_consts_path)
    else:
        matrix_consts = MatrixConsts()`
Same code in 2 places

Reply:
Done